packages/core/strapi/lib/commands/transfer/utils.js

```diff
@@ -45,15 +45,18 @@ const buildTransferTable = (resultData) => {
     totalItems += item.count;
 
     if (item.aggregates) {
-      Object.keys(item.aggregates).forEach((subkey) => {
-        const subitem = item.aggregates[subkey];
+      Object.keys(item.aggregates)
+        // Sort aggregate keys by size
```

Comment:
Do we actually want to do this? It depends what the user is using the table for. It might be better to sort by the names rather than size unless the purpose of the table is just to see the sizes. @marcoautiero thoughts?

Reply:
Oh, that's interesting, when I looked at the table I immediately thought that its first use would be to assess what kind of data is the largest in an app. But your use case also makes sense. What value does sorting by name brings to you in a transfer scenario?

Reply:
For me it's just the quickest way of seeing that certain data was actually transferred, or checking the size of a specific part of the data. And it would always stay in (more or less) the same order instead of changing.

Plus, are we keeping the aggregate data sorting consistent with the overall transfer stage sorting? It could be confusing if they're sorted differently.

Reply:
Having alphabetical ordering brings value to searchability. After all, in the CTB and in the content editor we already have the content types alphabetically ordered.

Reply:
Alright, I've reverted this change & added an alphabetical order sort